docs/02.API-REFERENCE.md

```diff
@@ -757,6 +757,42 @@ jerry_eval (const jerry_char_t *source_p,
 - [jerry_create_external_function](#jerry_create_external_function)
 - [jerry_external_handler_t](#jerry_external_handler_t)
 
+## jerry_run_all_enqueued_jobs
+
+**Summary**
+
+Run enqueued Promise jobs until the first thrown error or until all get executed.
+
```

Comment:
Please add a note too:
` *Note*: Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
 is no longer needed.`